src/csharp/Microsoft.Spark/Services/ConfigurationService.cs

```diff
@@ -20,6 +20,7 @@ internal sealed class ConfigurationService : IConfigurationService
             "spark.dotnet.worker.writeBufferSize";
 
         private const string DotnetBackendPortEnvVarName = "DOTNETBACKEND_PORT";
+        private const string DotnetBackendIPAddressEnvVarName = "DOTNETBACKEND_IP_ADDRESS";
```

Comment:
For new env vars, let's prefix with `DOTNET_SPARK`, this should be `DOTNET_SPARK_BACKEND_IP_ADDRESS`,